src/state/accountSelectors.ts

```diff
@@ -158,6 +158,16 @@ export const getSubaccountOpenOrders = createSelector([getSubaccountOrders], (or
   )
 );
 
+export const getOrderByClientId = (orderClientId: number) =>
```

Comment:
nit: dunno if I feel strongly because I guess it's pretty self explanatory but do we want to write a function descriptor (given that all the other functions in this file have one)